src/config/config.ts

```diff
@@ -1,12 +1,15 @@
 import * as dotenv from 'dotenv';
-dotenv.config();
+const result = dotenv.config();
+
+if (result.error) {
+  console.log('Please set up a .env file with values. See README.md for more info.');
+  process.exit(0);
+}
 
 export default {
-  jwrSecret: process.env.JWT_SECRET || 'seedSecreteToken',
+  jwrSecret: process.env.JWT_SECRET || ' ',
   DB: {
-    URI: process.env.DB_MONGO_DB_URL || 'mongodb://localhost:27017/petsLove-typescript',
-    USER: '',
-    PASSWORD: '',
+    URI: `mongodb://${process.env.MONGO_DB_USER}:${process.env.MONGO_DB_PASSWORD}@${process.env.MONGO_DB_HOST}:${process.env.MONGO_DB_HOST_PORT}/${process.env.MONGO_DB_NAME}?authSource=admin`,
```

Comment:
Please, change this line for this:
```
URI:
      process.env.URI_MONGO_DB_PROD ||
`mongodb://localhost:${process.env.DOCKER_DB_HOST_PORT}/${process.env.DOCKER_MONGO_DB_NAME}`,
```

Reply:
Hi! I'm glad you like the idea for my PR :)

I'm not sure how the production environment is set up. But can't you set the environment variables like this in prod?
```
MONGO_DB_PASSWORD=prodPassword
MONGO_DB_USER=prodUser
MONGO_DB_HOST=195.88.54.16 (example of prod IP)
MONGO_DB_HOST_PORT=27017
MONGO_DB_NAME=petsLove-typescript
```

Then it should work without the extra line of code.

Reply:
Thanks for asking. I am not an expert on the subject.
But, in prod I have a env URI_MONGO_DB_PROD with the that URL that mongoDB gave me and it's a litter different.

Maybe it could be like this:
`mongodb+srv://${MONGO_DB_USE}:${MONGO_DB_PASSWORD}@${MONGO_DB_HOST}/${MONGO_DB_NAME}?retryWrites=true&w=majority`

Do you think it's ok?

Reply:
I've updated the PR. 
The env file now contains an option for all components of the mongodb connection string: https://docs.mongodb.com/manual/reference/connection-string/#components
So you can now just split up the components and fill them in, f.ex. mongodb+srv:// as prefix and add the options at the end. 

I also changed the readme to say that the .env file should be setup before running docker-compose. If not, it will fail because the mongodb port value cannot be empty.